### PR TITLE
dockerize the example application

### PR DESCRIPTION
**Pull request by deanna-cooney**
The example third party application has been dockerized! 

The `readme.md` has had part of its setup instructions split into `docs/manual_setup.md` which contains the old setup instructions (plus a minor change due to the Docker configs) and `docs/docker_setup.md` which contains the new setup instructions for Docker. Both methods of setting up this repo still work. 

**How to test?**
- [ ] After following the `manual_setup` setup instructions, you can successfully log into Switchboard 
- [ ] After following the `docker_setup` setup instructions, you can successfully log into Switchboard

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# This is what John did in his Docusign fix.
+#      - "app.myclio.ca:${DOCKER_HOST_IP:-127.0.0.1}"
+
+# Apparently in Docker 20.10 (which i'm using) you can do the below instead of resolving the host manually.
+      - "host.docker.internal:host-gateway"
+
+# Both John's and my version both work the same, you can enter a username/password and accept oauth scopes before
+# you get the following error:
+#   HTTP::ConnectionError in ApplicationController#identity_callback
+#   failed to connect: Connection refused - connect(2) for "account.myclio.ca" port 1337
```
Comment (arian-kh) [inline docker-compose.yml:28]: This means Switchboard is unable to resolve `account.myclio.ca`, similar to how Docusign was unable to resolve `app.myclio.ca`. The same fix could help here.

**Review by JohnPark97 — APPROVED** [review]
LGTM 😁 
giphub

File: docs/manual_setup.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+On macOS installing the `postgresql` package is also required to be able to use the `psql` client command, as well as to install any libraries that are required to build the `pg` driver for use with `rails`.
+
+```
+brew install postgresql
+```
+
+By default, the database configuration is set up for use with Docker. For manual setup, edit `config/database.yml` and remove the following from `default`:
+
+```
+host: db
```
Comment (stevelyall) [inline docs/manual_setup.md]:
```suggestion
host: sb_db
```

Nitpick- this seems to be the current value of host in `database` I had to remove for manual config

File: docs/docker_setup.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+## Manual Setup
```
Comment (stevelyall) [inline docs/docker_setup.md]:
```suggestion
## Docker Setup
```

File: README.md [hunk 4]
```diff
@@ -28,59 +28,19 @@ Clio Identity is Clio's authentication service and identity provider. Similar to
...
-To get the example third-party application running on your local environment, ensure that you have valid API key pairs for both Clio Manage and Clio Identity, and that you have PostgreSQL, Ruby and Rails installed, as described in the Requirements section above.
-
-Clone the repository to your local environment:
-
-```
-git clone https://github.com/clio/example-third-party-application.git
-```
-
-Update the environment variables in `config/local_env.yml`:
+Then update the environment variables in `.env`:
```
Comment (stevelyall) [inline README.md]: I had to change `local_env.yml` for the app to load the environment variables properly. It doesn't seem to be loading from the `.env` instead.  I may have missed a step?

Are you intending to use `dotenv-rails` and `Dotenv.load` in the config?

**Review by stevelyall — CHANGES_REQUESTED** [review]
QA passes when the environment variables are set in the `local_env.yaml` 👍 

- [ ] `.env` not loading in a fresh setup? 

Made some small suggestions to the readmes, but otherwise looks good! 

Nice work!

File: README.md (hunk 4 above)
Comment (deanna-cooney) [reply README.md]: There is no `.env`, just missed updating this line in the readme.

**Review by stevelyall — APPROVED** [review]
QA Pass

**Pull request merged**